src/test/groovy/tests/DatadogTest.groovy

```diff
@@ -369,8 +343,8 @@ class DatadogTest extends DataAnalyticsSteps {
         def from_timestamp = (now.getTime()-1800000).toString().substring(0,10)
         def to_timestamp = (now.getTime()).toString().substring(0,10)
         def query = "sum:active_downloading_ips{*} by {remote_address}.as_count()"
-        Response response = datadog.datadogQueryTimeSeriesPoints(datadog_url,
-                datadog_api_key, datadog_application_key, from_timestamp, to_timestamp, query)
+        Response response = datadog.datadogQueryTimeSeriesPoints(datadogBaseURL,
+                datadogApiKey, datadogApplicationKey, from_timestamp, to_timestamp, query)
         def IPv4andIPv6Regex = "(^remote_address:)(([a-za-z:])([0-9A-Fa-f]{1,4}:){7}[0-9A-Fa-f]{1,4}|(\\d{1,3}\\.){3}\\d{1,3})"
```

Comment:
You have this regex defined over-and-over again. If you insist on using it then at least extract it to a single constant:
`private static constant ...`